app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -17,6 +17,15 @@ def process_async_state(current_async_state)
 
         if current_async_state.done?
           process_result(current_async_state.result)
+          @flow.analytics.track_event(
```

Comment:
If we're logging billing cost in `process_result`, would it make sense to put the new logging there as well?

Aside: Totally unrelated to this, but where is the definition of the `add_cost` method that's called in `process_result`? 🤔

Reply:
LOL, I hadn't since it was using just the result but, hey! I can change that 😝  Done in 349594d27

Reply:
> LOL, I hadn't since it was using just the result but, hey! I can change that 😝 Done in [349594d](https://github.com/18F/identity-idp/commit/349594d27c3ef8bac491ae95890d4b6c82a885f8)

Ah, I'd missed that the new behavior requires access to the `async_state`. Maybe a bit odd that we're now passing that as the argument to a function called "process _result_", but I think it'd still be fair to say the bulk of the method is still concerned with handling the result.